src/Sulu/Bundle/AdminBundle/Resources/public/scss/modules/login.scss

```diff
@@ -1,3 +1,19 @@
+// Colors
+$loginMediaContainerBackgroundColor: $grayDarkLighter !default;
+$loginMediaDarkenerBackgroundColor: $black !default;
+$loginContentContainerBackgroundColor: $blue !default;
+$loginMessageTextColor: $white !default;
+$loginHuskeyValidateTextColor: $red !default;
```

Comment:
`Husky` is misspelled here, and I think I would try to avoid using it as part of a name anyway.